R/ggplot.R

```diff
@@ -34,8 +34,9 @@ log10_reverse_trans <- function() {
 #' @description it perform logit scaling with right axis formatting. To not be used directly but with ggplot (e.g. scale_y_continuous(trans = "log10_reverse") )
 #'
 #' @importFrom scales label_scientific
+#' @importFrom scales trans_new
 #' @importFrom scales extended_breaks
-#' @importFrom stats qlogis plogis
+#' @importFrom functional Compose
 #'
```

Comment:
We would not add any dependency here, but check if `functional` is installed, and if not , error the user, asking for installing.

This should be applied to all cases.